docs/book/src/getting-started/testdata/project/internal/controller/memcached_controller_test.go

```diff
@@ -83,31 +81,25 @@ var _ = Describe("Memcached Controller", func() {
 			})
 			Expect(err).NotTo(HaveOccurred())
 			By("Checking if Deployment was successfully created in the reconciliation")
-			Eventually(func() error {
+			Eventually(func(g Gomega) {
```

Comment:
I can also remove this Eventually, since after reconciling, we actually _expect_ the deployment to have been created.  Should i do that too @camilamacedo86?